Specifications/Events/Store/Converters/for_AggregateEventToSDKConverter/given/committed_aggregate_events_and_a_converter.cs

```diff
@@ -31,7 +31,7 @@ public class committed_aggregate_events_and_a_converter : a_converter_and_a_prot
         event_source = "Rena Pearson";
         aggregate_root_id = "717915c1-bb88-4bec-b1c1-61451c5a6608";
 
-        aggregate_root_version = 186206759u;
+        aggregate_root_version = 0;
```

Comment:
Is there a particular reason why this version was changed?  Does it break something in this particular case?

Reply:
I don't remember, I just remember it being broken